src/styles/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$transitionDuration: 300ms;
```

Comment:
for CSS variables are better to use kebab-case (`$transition-duration`) to have consistency over all CSS.